.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: php
 
+env:
+  global:
+    - CACHE_DIR=$HOME/.mink-debug-cache
```

Comment:
I don't understand what do you mean by the default folder. `$PROJECT_CACHE_DIR`?